src/main/java/io/weaviate/client/v1/async/schema/api/VectorAdder.java

```diff
@@ -0,0 +1,96 @@
+package io.weaviate.client.v1.async.schema.api;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import org.apache.hc.client5.http.impl.async.CloseableHttpAsyncClient;
+import org.apache.hc.core5.concurrent.FutureCallback;
+import org.apache.hc.core5.http.ContentType;
+import org.apache.hc.core5.http.HttpResponse;
+
+import io.weaviate.client.Config;
+import io.weaviate.client.base.AsyncBaseClient;
+import io.weaviate.client.base.AsyncClientResult;
+import io.weaviate.client.base.Response;
+import io.weaviate.client.base.Result;
+import io.weaviate.client.base.http.async.ResponseParser;
+import io.weaviate.client.base.util.UrlEncoder;
+import io.weaviate.client.v1.auth.provider.AccessTokenProvider;
+import io.weaviate.client.v1.schema.model.WeaviateClass;
+import io.weaviate.client.v1.schema.model.WeaviateClass.VectorConfig;
+
+public class VectorAdder extends AsyncBaseClient<Boolean> implements AsyncClientResult<Boolean> {
+  private final ClassGetter getter;
+
+  private String className;
+  private Map<String, VectorConfig> addedVectors = new HashMap<>();
+
+  public VectorAdder(CloseableHttpAsyncClient client, Config config, AccessTokenProvider tokenProvider) {
+    super(client, config, tokenProvider);
+    this.getter = new ClassGetter(client, config, tokenProvider);
+  }
+
+  public VectorAdder withClass(String className) {
```

Comment:
I think that our convention is to name those methods `withClassName` not `withClass`